src/data/user.js

```diff
@@ -14,25 +14,22 @@ export function init() {
 }
 
 async function loadAuthenticatedUser() {
-  const user = await getUser()
+  const user = await GET('users/me')
 
   me.set(user)
 }
 
 export async function changeNickname(nickname) {
-  const mergedUpdates = Object.assign({}, get(me), { nickname })
-
-  const newUser = await updateUser(mergedUpdates)
+  const photo_id = get(me).photo_id
+  const newUser = await PUT('users/me', {nickname, photo_id})
```

Comment:
is newUser actually a newly created user or is updatedUser more accurate?

Reply:
`updatedUser` would be better